### PR TITLE
feat: Entrepot marketplace button for fully_funded projects

### PR DESCRIPTION
**Pull request by zaarheed**

### CODE REVIEW

**Comment by MaxLascombe** [comment]
I'll finalize the entrepot link storage in backend this weekend of monday.

**Review by MaxLascombe — APPROVED** [review]

File: pages/project.js [hunk 1]
```diff
@@ -67,13 +68,16 @@ export default function ProjectDetails() {
...
                 owner,
+                marketplace: marketplaceLinks,
             }
         },
         {
             refetchOnWindowFocus: false,
         }
     )
 
+    if (project) project.status = [{ fully_funded: null }]
```
Comment (zaarheed) [inline pages/project.js:79]: @MaxLascombe not sure what this bit is doing?

Would it not end up setting the project status to `fully_funded` for all projects?

Comment (MaxLascombe) [reply pages/project.js:79]: Ah, good catch. I was using that for testing purposes. I'll remove it now.

**Comment by MaxLascombe** [comment]
Ready to merge and deploy!

**Pull request merged**